hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/BaseActionExecutor.java

```diff
@@ -57,7 +57,7 @@ public BaseActionExecutor(HoodieEngineContext context, HoodieWriteConfig config,
    * @param metadata commit metadata of interest.
    */
   protected final void writeTableMetadata(HoodieCommitMetadata metadata) {
-    table.getMetadataWriter().ifPresent(w -> w.update(metadata, instantTime));
+    table.getMetadataWriter().ifPresent(w -> w.update(metadata, instantTime, false));
```

Comment:
hmmm. should we assume it's a table service at this level? The idea is that even if BaseActionExecutor is further extended, they don't end up scheduling this on metadata table.

Reply:
yeah, we can't assume so. will fix it